es6-wrapped-promise.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+
+module.exports = (Promise, ensureAslWrapper) => {
+  // Updates to this class should also be applied to the the ES3 version
+  // in index.js.
+  return class WrappedPromise extends Promise {
+    constructor(executor) {
+      var context, args;
+      super(wrappedExecutor);
+      var promise = this;
+
+      try {
+        executor.apply(context, args);
+      } catch (err) {
+        args[1](err)
+      }
+
+      return promise;
+      function wrappedExecutor(resolve, reject) {
+        context = this;
+        args = [wrappedAccept, wrappedReject];
```

Comment:
An effort was made recently to rename all `accept` to `resolve`. A few places this was missed, but I made a PR to address this: #108

I didn't have commit access at the time, otherwise I could probably just have merged it my self as it's just naming. It have been sitting there for over a month now with no comments so I took the liberty to just merge it now. I hope that's ok 😅